RUFAS/data_validator.py

```diff
@@ -201,21 +199,24 @@ def validate_properties(metadata: Dict[str, Any], metadata_depth_limit: int) ->
         deepest_path: list[str] = []
 
         type_to_validator_map: Dict[str, Callable[[list[str], dict[str, Any]], tuple[bool, str]]] = {
-            "number": DataValidator._metadata_number_validator,
-            "array": DataValidator._metadata_array_validator,
-            "bool": DataValidator._metadata_bool_validator,
-            "string": DataValidator._metadata_string_validator,
-            "object": DataValidator._metadata_object_validator,
+            "number": self._metadata_number_validator,
+            "array": self._metadata_array_validator,
+            "bool": self._metadata_bool_validator,
+            "string": self._metadata_string_validator,
+            "object": self._metadata_object_validator,
         }
 
         while stack:
             current_obj, depth, path = stack.pop()
 
             if depth > metadata_depth_limit:
-                om.add_error(
-                    "Max metadata depth exceeded.",
-                    f"Metadata depth exceeds maximum allowed depth of {metadata_depth_limit} at path {path}",
-                    info_map,
+                self.event_logs.append(
+                    {
+                        "error": "Max metadata depth exceeded.",
+                        "error message": f"Metadata depth exceeds maximum allowed depth"
```

Comment:
```suggestion
                        "error_message": f"Metadata depth exceeds maximum allowed depth"
```
For consistency with the rest of the codebase